File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Stores/IStoreData.cs
﻿using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Stores
{
    public interface IStoreData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IStrorePhone, IStroreEMail, IStroreFax
    {
        IDataItemList<IDataItemData> Dataset { get; set; }

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Stores/IStoreData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Stores;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IStoreData extends IStoreEmail,IStoreFax,IStorePhone, CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
